### PR TITLE
mozilla-ford-nlnet

### PR DESCRIPTION
**Pull request by teolemon**
mozilla-ford (draft)

### CODE REVIEW

File: lang/en/texts/partners.html [hunk 1]
```diff
@@ -68,6 +68,20 @@ <h2>Scientific collaborations</h2>
...
+      <a href="https://"><img src="https://static.openfoodfacts.org/images/misc/mozilla-foundation-logo.png" alt="Mozilla Open Source Support Program (MOSS)"></a>
+      <a href="https://"><img src="https://static.openfoodfacts.org/images/misc/opencollective-logo.png" alt="Open Collective"></a>
+        </div>
+
+        <div class="small-7 medium-12 large-8 xlarge-9 columns">
+            <p>The development of the new Open Food Facts cross-platform mobile application is sponsored thanks to the generous award of the <a href="https://www.fordfoundation.org/">Ford Foundation</a> and <a href="https://www.mozilla.org/en-US/moss/">Mozilla Open Source Support Program (MOSS)</a>, with the support of <a href="https://www.opencollective.com/">Open Collective</a>.</p>
 
+            <p><b>4-22-2020:</b> <a href="https://en.blog.openfoodfacts.org/mozilla-and-the-ford-foundation-support-the-new-open-food-facts-app/">Open Food Facts gets an award by the Mozilla and Ford foundations for a new mobile app for even greater impact.</a></p>
```
Comment (teolemon) [inline lang/en/texts/partners.html:84]:
```suggestion
        </div>
    </div>

<div class="row">
        <div class="small-5 medium-12 large-4 xlarge-3 columns">
            <a href="https://www.fordfoundation.org/"><img src="https://static.openfoodfacts.org/images/misc/ford_logo.svg" alt="Ford Foundation"></a>
      <a href="https://"><img src="https://static.openfoodfacts.org/images/misc/mozilla_logo.svg" alt="Mozilla Open Source Support Program (MOSS)"></a>
      <a href="https://"><img src="https://static.openfoodfacts.org/images/misc/opencollective_logo.svg" alt="Open Collective"></a>
        </div>  

        <div class="small-7 medium-12 large-8 xlarge-9 columns">
            <p>The development of the new Open Food Facts cross-platform mobile application is sponsored thanks to the generous award of the <a href="https://www.fordfoundation.org/">Ford Foundation</a> and <a href="https://www.mozilla.org/en-US/moss/">Mozilla Open Source Support Program (MOSS)</a>, with the support of <a href="https://www.opencollective.com/">Open Collective</a>.</p>

            <p><b>4-22-2020:</b> <a href="https://en.blog.openfoodfacts.org/mozilla-and-the-ford-foundation-support-the-new-open-food-facts-app/">Open Food Facts gets an award by the Mozilla and Ford foundations for a new mobile app for even greater impact.</a></p>
```

File: lang/en/texts/partners.html [hunk 2]
```diff
@@ -68,6 +68,19 @@ <h2>Scientific collaborations</h2>
         <div class="small-7 medium-12 large-8 xlarge-9 columns">
             <p><a href="https://etude-nutrinet-sante.fr/">NutriNet-Santé</a> is a cohort study that tracks the nutrition and health of volunteers for several years. EREN uses NutriNet data cross-referenced with Open Food Facts data to study the long-term effects of food additives.</p>
             <p><b>→</b> <a href="https://etude-nutrinet-sante.fr/profil/introduction">Participate in Nutri-Net</a> to help research (For French residents).</p>
+        </div>
+    </div>
+
+<div class="row">
+        <div class="small-5 medium-12 large-4 xlarge-3 columns">
+            <a href="https://www.fordfoundation.org/"><img src="https://static.openfoodfacts.org/images/misc/ford_logo.svg" alt="Ford Foundation"></a>
+      <a href="https://"><img src="https://static.openfoodfacts.org/images/misc/mozilla_logo.svg" alt="Mozilla Open Source Support Program (MOSS)"></a>
```
Comment (teolemon) [inline lang/en/texts/partners.html:77]:
```suggestion
      <a href="https://www.mozilla.org/en-US/moss/"><img src="https://static.openfoodfacts.org/images/misc/mozilla_logo.svg" alt="Mozilla Open Source Support Program (MOSS)"></a>
```

File: lang/en/texts/partners.html [hunk 3]
```diff
@@ -68,6 +68,19 @@ <h2>Scientific collaborations</h2>
...
+      <a href="https://"><img src="https://static.openfoodfacts.org/images/misc/opencollective_logo.svg" alt="Open Collective"></a>
```
Comment (teolemon) [inline lang/en/texts/partners.html:78]:
```suggestion
      <a href="https://opencollective.com/open-source-speed-dating/events/open-source-speed-dating-fosdem-2020-96f78c6c"><img src="https://static.openfoodfacts.org/images/misc/opencollective_logo.svg" alt="Open Collective"></a>
```

**Comment by stephanegigandet** [comment]
Could you add Nl-Net too?

**Pull request merged**